SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialStaticComponentView.h

```diff
@@ -2,8 +2,6 @@
 
 #pragma once
 
-#include "CoreMinimal.h"
```

Comment:
I guess you'd still need `#include "CoreMinimal.h"` as long as it's a UObject?

Reply:
Will add UObject, Map + UniquePtr include, CoreMinimal seems excessive